datalad_next/create_sibling_webdav.py

```diff
@@ -108,13 +108,12 @@ class CreateSiblingWebDAV(Interface):
         ),
         existing=Parameter(
             args=("--existing",),
-            constraints=EnsureChoice('skip', 'error', 'reconfigure', None),
+            constraints=EnsureChoice('skip', 'error', 'reconfigure'),
             doc="""action to perform, if a (storage) sibling is already
-            configured under the given name and/or a target already exists.
-            In this case, a dataset can be skipped ('skip'), an existing target
-            repository be forcefully re-initialized, and the sibling
-            (re-)configured ('reconfigure'), or the command be instructed to
-            fail ('error').""", ),
+            configured under the given name.
+            In this case, a sibling creation can be skipped ('skip') or the
+            sibling (re-)configured ('reconfigure'), or the command be
+            instructed to fail ('error').""", ),
```

Comment:
What if the target already exists? The condition is removed from the docs, does it mean it won't be handled?

Reply:
Yes. I don't think it can be handled. How would we discover it and what would we need to to differently in such case?

Reply:
It could offer to verify that the remote state actually matches the local expectations. Here is a function that implemented that (I mentioned it in the chat earlier).

https://github.com/mih/datalad-mihextras/blob/6245ed3192040624c7db7d25c1fcf10e4b49976d/datalad_mihextras/export_to_webdav.py#L243-L271

The key point is: What is a user supposed to do, if there is conflicting content on the remote? The documentation is now silent about this -- which is suboptimal even if it turns out that really nothing can be done (which I am not convinced of).

Reply:
Conflicting what with? This is not push, but create.

Reply:
I think the only option is to say: We talk directly (not via special remote) to the webdav location to find whether there's anything at all and consider this `existing`.

Reply:
For now just added respective entry to the command's doc.

Reply:
Left a comment there